ChangeLog

```diff
@@ -1,5 +1,15 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x released xxxx-xx-xx
+
+Changes
+   * Added config.h option MBEDTLS_NO_UDBL_DIVISION, to prevent the use of
+     64-bit division.
+
```

Comment:
This should be formulated independent of whether the architecture is 32-bit or 64-bit.

My main point about the renaming of `MBEDTLS_NO_UDBL_DIVISION` to `MBEDTLS_NO_UDBL` - or perhaps `MBEDTLS_MPI_NO_UDBL` - is that I want to allow the user to disable the use of double-width integer types regardless of the reason. In particular, I wouldn't document `MBEDTLS_NO_UDBL_XXX` as "The platform lacks support for double-width integer division" but rather as "Don't use double-width integer types in the MPI library." and go on by explaining why one might want to do that, e.g. because there's no such type, or just for testing. That's also more in line with, e.g., `MBEDTLS_HAVE_INT32` which the user might want to pick even if the architecture is 64-bit (... and for which I'd prefer the name `MBEDTLS_MPI_USE_INT32/64`, but that's yet another discussion).